tools/cldr-code/src/main/java/org/unicode/cldr/tool/ChartGrammaticalForms.java

```diff
@@ -966,12 +967,16 @@ public static String getBestBaseUnit(
             Rational bestFactor = null;
             Rational inputBoundary = Rational.of(2).multiply(info.factor);
             for (Entry<Rational, String> entry : factorToUnit.entrySet()) {
+                final String possibleUnit = entry.getValue();
+                if (possibleUnit.equals("cup-jp")) {
+                    continue; // skip odd unit
+                }
                 final Rational currentFactor = entry.getKey();
```

Comment:
does this need a ticket for followup?

Reply:
Good question. This particular line doesn't, but there should be a change to favor the regular metric units. Will do.